python_cmake_module/cmake/Modules/FindPythonExtra.cmake

```diff
@@ -211,7 +222,11 @@ if(PYTHONINTERP_FOUND)
   endif()
 
   if(WIN32)
-    set(PythonExtra_EXTENSION_EXTENSION ".pyd")
+    if("${CMAKE_BUILD_TYPE} " STREQUAL "Debug ")
+      set(PythonExtra_EXTENSION_EXTENSION "_d.pyd")
```

Comment:
Not necessarily, but if one doesn't have a debug interpreter, Visual Studio won't be able to link against it and the compilation will fail. So the suffix of the extensions would not matter in that case.